spec/serializer_spec.rb

```diff
@@ -847,4 +847,24 @@ def serialize_primary(object, options = {})
       })
     end
   end
+
+  describe 'include validation' do
+    it 'raises an exception when join character is invalid' do
+      expect do
+        JSONAPI::Serializer.serialize(create(:post), include: 'long_comments');
+      end.to raise_error(JSONAPI::Serializer::InvalidIncludeError, "'long_comments' is not a valid include.  Did you mean 'long-comments' ?")
```

Comment:
Looks good! Just drop the explicit message checks here and below — I consider `raise_error(JSONAPI::Serializer::InvalidIncludeError)` to be a strong enough test but not too strong, and it keeps the lines under 100 characters.